sdks/python/apache_beam/io/hadoopfilesystem_test.py

```diff
@@ -323,7 +375,7 @@ def test_create_success(self):
     url = self.fs.join(self.tmpdir, 'new_file')
     handle = self.fs.create(url)
     self.assertIsNotNone(handle)
-    url = self.fs._parse_url(url)
+    _, url = self.fs._parse_url(url)
```

Comment:
Test `server` is None, here and below?

Reply:
There will be a separate `test_parse_url` to test these return values.